utils/linux.go

```diff
@@ -180,6 +183,32 @@ func KernelCurrentVersionCmp(k1Major, k1Minor int) (int, error) {
 	return -1, nil
 }
 
+// Parses the kernel release string (obtained from GetKernelRelease()) and returns
```

Comment:
Don't think this function is needed now that we have `KernelCurrentVersionCmp()` above.

Reply:
I still think the function is useful as a way to parse kernel version numbers (i.e., one may want to parse them for reasons other than comparing them against the current kernel version).